packages/main/src/plugin/index.ts

```diff
@@ -120,15 +123,52 @@ export class PluginSystem {
     });
   }
 
+  // Create an Error to access stack trace
+  // Match a regex for the file name and return it
+  // return nothing empty string if file name not found
+  async getExtName(){
+
+    const stack = new Error().stack
+
+    var validExt:string[] = [];
+
+    if(this.extensionLoader != undefined){
+
+      //Take the id information from extensionLoader
+      let validExtObj = await this.extensionLoader.listExtensions();
+
+      validExtObj.forEach((extInfo: any) => {
+        validExt.push(extInfo.id);
```

Comment:
I think we should grab the path of the extension (add that information in the info object)
so we don't hardcode some path in the regexp